sql/catalyst/src/test/scala/org/apache/spark/sql/connector/InMemoryTableCatalog.scala

```diff
@@ -30,12 +30,9 @@ import org.apache.spark.sql.sources.v2.Table
 import org.apache.spark.sql.types.StructType
 import org.apache.spark.sql.util.CaseInsensitiveStringMap
 
-class InMemoryTableCatalog extends TableCatalog with SupportsNamespaces {
+class InMemoryTableCatalogBase extends TableCatalog {
```

Comment:
I tried to have a catalog without `SupportsNamespace`. If you have a better name, please let me know. :)

Reply:
fine to me

Reply:
maybe `BasicInMemoryTableCatalog`

Reply:
Changed to `BasicInMemoryTableCatalog`.